### DISCUSSION TITLE
Guide Me on the Project.

### DISCUSSION THREAD

**Discussion by amitpant7** [fastmachinelearning/hls4ml#813] [General]
I have to do a project ' Hardware aware neural architectural search on FPGA'. What process should we follow and what could be the possible learning path? and are there any good tutorials on using HLS4ML end-to-end?

**Comment by pierinim** [comment]
Hello. 

It’s a long path… Some of us have plans to kickoff a dedicated project on this, provided funds. We are now working on the funds. We might actually open dedicated jobs and student internships on this by the end of the year, it things work. 

There was some first attempt in a paper we wrote with the QKeras team

https://arxiv.org/abs/2006.10159

not yet NAS and not yet custom on FPGAs. We plan to build from there

Maurizio
